client/components/forms/docs/example.jsx

```diff
@@ -35,7 +36,10 @@ import PhoneInput from 'components/phone-input';
 /**
  * Internal dependencies
  */
-var countriesList = require( 'lib/countries-list' ).forSms();
+const countriesList = require( 'lib/countries-list' ).forSms();
+const currencies = require( 'lib/format-currency/currencies' ).CURRENCIES
```

Comment:
Note, that this isn't a blocker 😄 but a stray codemod might try and modify this to import usage in the future.